motr/m0crate/scripts/gen_index_yaml_run_workload

```diff
@@ -0,0 +1,89 @@
+#	script: m0crate/scripts/gen_index_yaml_run_workload
```

Comment:
Please add
```
#!/bin/bash
```
as the first line of a shell script.

Reply:
Any new file (maybe except the README.md) should have a Seagate Copyright header.
Please refer to other files in the source code tree, and add proper copyright header here.

Reply:
Done.